### PR TITLE
Update to 1.12.1

### PR DESCRIPTION
**Pull request by sumit0190**
This is a brand-new feedstock, so the recipe was crafted from scratch. Intel needs this version (1.12.1) as soon as possible.

### CODE REVIEW

**Review by ELundby45 — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+source:
+  url: https://github.com/intel/{{ name }}/archive/refs/tags/v{{ version }}00.tar.gz
+  sha256: 6f2b54fc3e0a5e26ccaa0552ba93e6d7ecbdc2902b4920e86fde5efdfe427ebc
+  patches:
+    - patches/0001-add-git-rev.patch
+
+build:
+  number: 0
+  # Anything other than linux-64 is not supported fully by upstream, and 3.11
+  # doesn't have PyTorch yet.
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: This link https://intel.github.io/intel-extension-for-pytorch/latest/tutorials/installation.html#install-via-wheel-file will help our future selves

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    - libllvm14 14.0.6
+    - pytorch {{ version }}
+
+test:
+  requires:
+    - pip
+    - torchvision
+  commands:
+    - pip check
+    - python -c "import torch; import torchvision; import intel_extension_for_pytorch"
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: I would modified it as:
`python -c "import torch; import torchvision; import intel_extension_for_pytorch as ipex; print('torch:', torch.__version__,' ipex:',ipex.__version__)"`
see https://intel.github.io/intel-extension-for-pytorch/latest/tutorials/installation.html#build-docker-container-from-dockerfile

File: recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    # Both OpenMP installations can be used with this package (can be selected at runtime).
+    - intel-openmp
+    - _openmp_mutex
+    - libllvm14 14.0.6
+    - pytorch {{ version }}
+
+test:
+  requires:
+    - pip
+    - torchvision
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `torchvision` can be removed as it's already in `run`

File: recipe/meta.yaml (hunk 1 above)
Comment (akabanovs) [reply recipe/meta.yaml]: Thanks, I added to the comments.

File: recipe/meta.yaml (hunk 3 above)
Comment (akabanovs) [reply recipe/meta.yaml]: Yes, thanks.

File: recipe/meta.yaml (hunk 2 above)
Comment (akabanovs) [reply recipe/meta.yaml]: Thanks.

File: README.md [hunk 4]
```diff
@@ -1 +1,15 @@
-# intel-extension-for-pytorch-feedstock
```
Comment (M-Waszkiewicz-Anaconda) [inline README.md:1]: I would leave the file untouched

File: recipe/build.sh [hunk 5]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -ex
+
+# This is silly, but Intel needs a specific commit of their oneDNN stuff.
+pushd third_party/llga
+git init
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/build.sh:6]: I would leave a comments what is going on here

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
overall LGTM, just describe what we are doing in build.sh for further reference

**Pull request merged**